contracts/package.json

```diff
@@ -62,5 +62,10 @@
         "typechain": "^5.1.2",
         "typescript": "^4.5.5"
     },
-    "dependencies": {}
+    "dependencies": {
+        "@walletconnect/web3-provider": "^1.7.3",
+        "eth-provider": "^0.10.0",
+        "react-app-rewired": "^2.2.1",
+        "walletlink": "^2.5.0"
+    }
```

Comment:
These got added to the `/contracts` package.json - they should be in the `/frontend` one.

Reply:
oh fook my bad LOL

Reply:
removed